src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -71,6 +75,66 @@ public void RunBenchmarks(string assemblyPath)
             s_profiler(assemblyPath, Configuration.RunId, _outputDirectory, runner, collectOutputFilesCallback);
         }
 
+        public void RunStartupBenchmark(string[] args, ProcessStartInfo psi, PreDel pre, MidDel mid, PostDel post)
+        {
+            pre(_performanceTestConfig, psi);
+            int iterations = _performanceTestConfig.iterations;
+            int timeout = _performanceTestConfig.timeout;
+            psi.RedirectStandardOutput = true;
+
+            // Warmup run.
+            using (var p = new Process()) 
+            {
+                p.StartInfo = psi;
+                p.Start();
+                if (p.WaitForExit(timeout) == false) 
+                {
+                    if (p != null)
+                    {
+                        p.Kill();
+                    }
+                    Console.WriteLine("Timeouted!");
```

Comment:
>Console.WriteLine [](start = 20, length = 17)

In general, please use the `PerformanceLogger` class instead of Console.WriteLine, as we can control the API output with a verbosity flag in the future.